app/script/conversation/ConversationRepository.js

```diff
@@ -1837,22 +1837,22 @@ z.conversation.ConversationRepository = class ConversationRepository {
    * @returns {Promise} Resolves when sent status was updated
    */
   _update_message_as_sent(conversation_et, message, event_time) {
-    return this.get_message_in_conversation_by_id(conversation_et, message.id)
-      .then((message_et) => {
-        const changes = {
-          status: z.message.StatusType.SENT,
-        };
-        message_et.status(z.message.StatusType.SENT);
-
-        if (event_time) {
-          message_et.timestamp(new Date(event_time).getTime());
-          changes.time = event_time;
-        }
+    if (z.event.EventTypeHandling.STORE.includes(message_et.type) || message_et.has_asset_image()) {
```

Comment:
`message_et` needs to become `message` in `if (z.event.EventTypeHandling.STORE.includes(message_et.type) || message_et.has_asset_image())`?!